src/main/java/org/fundacionjala/at15/katas/calculator/Calculator.java

```diff
@@ -5,4 +5,12 @@
     public int doOperation(Operation operation, int leftOperand, int rightOperand) {
         return operation.run(leftOperand, rightOperand);
     }
+
+    public int doOperation(Operation operation, int[] operant) {
```

Comment:
typo, operand*. Also, since this is defined as an array, it would make more sense to rename the variable to `operands`